src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/TasksOperations.java

```diff
@@ -49,7 +49,7 @@ public boolean updateTask(TaskRecord taskRecord) throws IllegalArgumentException
      * @param experimentId the primary key of the experiment
      * @return the found task or empty if not found
      */
-    public Optional<TaskRecord> searchTask(String platform, int experimentId) {
+    public Optional<TaskRecord> findTask(String platform, int experimentId) {
```

Comment:
i have read something about find being one and search being multiple items. I don't think it's that clearly defined since most of the apis i remember use find (including the all ubiquitous java regex-api). But i think i was mislead by the original name. Consistent would be get ;) wait for a new commit